terraform/route53.tf

```diff
@@ -0,0 +1,83 @@
+#====================================================
+# Route 53
+#====================================================
+resource "aws_route53_zone" "whoogle_dns" {
+  name = var.domain_name
+}
+
+# resource "namecheap_domain_records" "whoogle_dns" {
```

Comment:
you can also move this parts to a sub directory and import it as a module.

if you do not need it you can just disable the import of the module